components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -659,6 +621,9 @@ constexpr char kEthereumPrefKey[] = "ethereum";
 constexpr char kFilecoinPrefKey[] = "filecoin";
 constexpr char kSolanaPrefKey[] = "solana";
 
+std::vector<mojom::BlockchainToken>& GetWyreBuyTokens();
```

Comment:
Safer to return as const ref

Reply:
Fixed 45fb06dd107ad0f973e6965959663ff299cc3f49